tests/store/artifact/test_mlflow_artifact_repo.py

```diff
@@ -8,11 +8,13 @@
 from mlflow.exceptions import MlflowException
 
 
-@pytest.fixture(scope="module", autouse=True)
-def set_tracking_uri():
+@pytest.fixture(
+    scope="module", autouse=True, params=["http://localhost:5000", "http://localhost:5000/"]
+)
```

Comment:
This doubles the number of tests. Can we just test that both `http://localhost:5000` and `http://localhost:5000/` can be resolved correctly?

Reply:
definitely!

Reply:
Created another fixture and used an explicit single test for validating alternate uri naming convention